arrow/arrow.py

```diff
@@ -537,7 +540,14 @@ def span(
             >>> arrow.utcnow().span('day', bounds='[]')
             (<Arrow [2013-05-09T00:00:00+00:00]>, <Arrow [2013-05-10T00:00:00+00:00]>)
 
+            >>> arrow.utcnow().span('week')
+            (<Arrow [2021-02-22T00:00:00+00:00]>, <Arrow [2021-02-28T23:59:59.999999+00:00]>)
+
+            >>> arrow.utcnow().span('week', week_start=6)
+            (<Arrow [2021-02-20T00:00:00+00:00]>, <Arrow [2021-02-26T23:59:59.999999+00:00]>)
         """
+        if not 1 <= week_start <= 7:
+            raise ValueError("condition 1 <= week_start <= 7 not met")
```

Comment:
Let's make this message a bit more readable like "Week start argument must be between 1 and 7."

Reply:
Changed text to "week_start argument must be between 1 and 7."